test/acceptance/framework/config/config.go

```diff
@@ -40,3 +48,90 @@ func (t TestConfig) TFVars(ignoreVars ...string) map[string]interface{} {
 	}
 	return vars
 }
+
+// UnmarshalTF populates the cfg struct with the Terraform outputs
+// from the given tfDir directory. The cfg arg must be a pointer to
+// a value that can be populated by json.Unmarshal based on the output
+// of the `terraform output -json` command.
+func UnmarshalTF(tfDir string, cfg interface{}) error {
```

Comment:
Yes currently it is only used by the HCP tests. I can be explicit about the type of the input arg, in which case it may also be best to move it into the `hcp` test package. I made this func generic and included it in the `config` package with the intention of eventually refactoring the `suite` package to use it.. I didn't want to make those changes under this PR because I didn't want to make non-functional changes to tests that I didn't change.

Let me know what you think.

Reply:
For now I've moved the HCP-specific config into a new `config.go` file in the `hcp` package. It explicitly uses the `HCPTestConfig` struct now.